### ISSUE TITLE
[Topic] Write a meaningful link text

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#110]
URL: https://wpaccessibility.org/docs/topics/content/links/link-text/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/links/link-text.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

- this content should be reviewed and restructured.
- the links should be checked and updated. More recent links could be added.
